src/components/EmailSignup/EmailSignup.tsx

```diff
@@ -0,0 +1,153 @@
+import { useReducer, FocusEvent, useRef, useEffect } from 'react';
+import {
+	Box,
+	Flex,
+	Button,
+	Input,
+	Text,
+	Spinner,
+	FlexProps,
+} from '@chakra-ui/react';
+import { AppEvents, initialState, reducer } from './reducer';
+import { emailRegex } from 'src/utils/validation';
+import { MARKETING_URI } from 'src/constants/urls';
+import localStore from 'src/utils/localStore';
+
+interface EmailSignupProps extends FlexProps {
+	page?: string;
+}
+
+export const EmailSignup = ({ page = 'home', ...props }: EmailSignupProps) => {
+	const [state, dispatch] = useReducer(reducer, initialState);
+	const inputRef = useRef<HTMLInputElement>(null);
+
+	const { submitted, errorMessage, valid, loading } = state;
+
+	useEffect(() => {
+		const previouslySubmitted =
+			localStore.get('marketing-form-submission') === 'true';
```

Comment:
hmm localStore seems to be doing JSON.parse. `JSON.parse('true') === true`, so this would never be true?

Reply:
Good catch